sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -187,6 +188,20 @@ public ServiceBusClientBuilder connectionString(String connectionString) {
         return credential(properties.getEndpoint().getHost(), tokenCredential);
     }
 
+    /**
+     * Enable cross entity transaction on the connection to Service bus. Use this feature when your transaction is
+     * involved in different Service Bus entities. You should avoid using non transaction activities on the clients
+     * which have this feature enabled.
```

Comment:
It's not clear why the user should avoid using non-transaction activities. The documentation should clearly describe when it is not suitable to do non-transactional activities on this client.